int32 position_x
int32 position_y
---
bool status
